components/contact/ContactForm.jsx

```diff
@@ -1,14 +1,36 @@
 import React, { useState, useEffect, useCallback } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { toast } from 'react-toastify';
+import { makeStyles } from '@material-ui/core/styles';
 import Grid from '@mui/material/Grid';
 import Button from '@mui/material/Button';
 import TextField from '@mui/material/TextField';
+import InputLabel from '@mui/material/InputLabel';
 import CircularProgress from '@mui/material/CircularProgress';
 import { showFeedbackSuccess, setErrorModal } from '@reducers/ui';
 import { sendGitRequest } from '@reducers/data';
 import 'react-toastify/dist/ReactToastify.css';
 
+const useStyles = makeStyles(theme => ({
+  form: {
+    marginBottom: theme.spacing(20)
+  },
+  formLabel: {
+    fontSize: '18px',
+    color: '#29404F',
```

Comment:
colors should either be added to the global theme or refer to an already existing color especially because of implications with dark mode

Reply:
I went ahead and set color from colors theme